### PR TITLE
Regenerate EventGrid Data Plane SDK to include recently added event types

### PR DESCRIPTION
**Pull request by kalyanaj**
Regenerate EventGrid Data Plane SDK to include recently added event types.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by kalyanaj** [comment]
@daschult , can you please help review/approve this PR?

File: lib/services/eventgrid/package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
   "license": "MIT",
   "main": "./lib/eventGridClient.js",
   "types": "./lib/eventGridClient.d.ts",
-  "homepage": "https://github.com/azure/azure-sdk-for-node/tree/master/lib/services/eventgrid",
+  "homepage": "https://github.com/azure/azure-sdk-for-node/lib/services/eventgrid",
```
Comment (amarzavery) [inline lib/services/eventgrid/package.json]: Looks like a bug in the generator. @kalyanaj - Can you manually revert the change on this line for now.

Comment (kalyanaj) [reply lib/services/eventgrid/package.json]: Thanks @amarzavery for the review, I have manually reverted this change on this line and have pushed the changes.
